OpenRA.Mods.Common/Scripting/Properties/PlayerProperties.cs

```diff
@@ -52,6 +55,12 @@ public string Race
 		[Desc("Returns true if the player is a bot.")]
 		public bool IsBot { get { return Player.IsBot; } }
 
+		[Desc("Returns true if the player is non combatant.")]
+		public bool IsNonCombatant { get { return Player.NonCombatant; } }
+
+		[Desc("Returns true if the player is the local player.")]
+		public bool IsLocalPlayer { get { return Player.PlayerActor.Owner == (Player.World.RenderPlayer ?? Player.World.LocalPlayer); }	}
```

Comment:
eh... I'm pretty sure that `Player` and `Player.PlayerActor.Owner` are the same.